api/CSSSupportsRule.json

```diff
@@ -5,57 +5,54 @@
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/CSSSupportsRule",
         "support": {
           "chrome": {
-            "version_added": false
+            "version_added": "41"
           },
           "chrome_android": {
-            "version_added": false
+            "version_added": "41"
           },
           "edge": {
-            "version_added": "≤18",
-            "version_removed": "79"
+            "version_added": "12"
           },
           "firefox": {
-            "version_added": "17",
-            "notes": "From Firefox 17 to 19, methods and properties were defined on <code>CSSSupportsRule</code>. From version 20, they were on <code>CSSConditionRule</code>.",
-            "flags": [
-              {
-                "type": "preference",
-                "name": "layout.css.supports-rule.enable",
-                "value_to_set": "true"
-              }
-            ]
+            "version_added": "22",
+            "notes": "Methods and properties are defined on <code>CSSConditionRule</code>."
```

Comment:
This doesn't need a note, it's the way it should be per https://drafts.csswg.org/css-conditional/#the-csssupportsrule-interface.

However, if the interface was exposed in Firefox 17, then I think we should keep that. Whether to include a note about the members is debatable, but I'd probably say no, a note about more things being supported than is represented in the data isn't really useful.